lewis/core/control_client.py

```diff
@@ -73,16 +73,30 @@ class ControlClient(object):
     of objects at the top level, a dictionary of named objects can be
     obtained via get_object_collection.
 
-    :param host: Host the control server is running on
-    :param port: Port on which the control server is listening
+    If a timeout is supplied, all underlying network operations time out
+    after the specified time (in milliseconds). The default is 1000 ms,
+    for no timeout specify ``None``.
+
+    :param host: Host the control server is running on.
+    :param port: Port on which the control server is listening.
+    :param timeout: Timeout in milliseconds for ZMQ operations.
     """
 
-    def __init__(self, host='127.0.0.1', port='10000'):
+    def __init__(self, host='127.0.0.1', port='10000', timeout=1000):
```

Comment:
I think it's okay to have a default, we could make it longer. 1 second was just some arbitrary number...like 3. I'm happy to change it to 3.